src/libcore/atomic.rs

```diff
@@ -1089,3 +1090,23 @@ pub fn fence(order: Ordering) {
         }
     }
 }
+
+macro_rules! impl_Debug {
+    ($($t:ident)*) => ($(
+        #[stable(feature = "atomic_debug", since = "1.3.0")]
+        impl fmt::Debug for $t {
+            fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+                f.debug_tuple(stringify!($t)).field(&self.load(Ordering::Relaxed)).finish()
```

Comment:
I agree with @nagisa, this should be `SeqCst` to take the most conservative route here. Orderings should only be relaxed if you really know what you're doing and have a global understanding of how an atomic variable is being accessed, otherwise you should be as conservative as possible and use `SeqCst` (e.g. this case)